samples/valueentity-counter/pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example</groupId>
+  <artifactId>valueentity-counter</artifactId>
+  <version>0.1</version>
+  <packaging>jar</packaging>
+
+  <name>first-service</name>
+    <properties>
+
+    <!-- TODO Update to your own Docker repository or Docker Hub scope -->
+    <dockerImage>my-docker-repo/${project.artifactId}</dockerImage>
```

Comment:
this is a simple suggestion for the future:

```xml
${env.DOCKER_REGISTRY}
```

Reply:
This looks good and we should change to it. But maybe not on this PR, let's move on and improve this later